nio.conf

```diff
@@ -1,103 +1,276 @@
+#
+#publishers_notification_timeout=5
+
+# time to wait after publisher is marked for deletion and it's actual deletion - allows for listeners to unsubscribe before publisher deletion
+#
+#listeners_removal_timeout=2
+
+# interval at which to retry removing listeners if some are still active after publisher deletion
+#
+#termination_retry_interval=0.1
+
+# time to wait before retrying a networking connection after failure
+#
+#connect_interval=1
+
+# time to wait when establishing a network connection
+#
+#connect_timeout=0
+
+# linger period determines how long pending messages which have yet to be sent to a peer shall linger in memory after a socket is disconnected
+#
+#linger=1000
+
+# high water mark is a hard limit on the maximum number of outstanding messages ØMQ shall queue in memory
+#
+#hwm_size=100000
+
+# sets the maximum number of sockets allowed on the context
+#
+#max_sockets=1023
+
+# size of the ØMQ thread pool to handle I/O operations. If your application is using only the inproc transport for messaging you may set this to zero, otherwise set it to at least one.
+#
+#max_io_thread_count=1
+
+# zmq setting timeout setting for initial client subscription
+#
+#zmq_port_reconnect_timeout=3
+
+[persistence]
+
+# directory to store persistence files
+#
+#data=etc/persist
+
+[scheduler]
+# minimum amount of time allowed for a scheduled job
+#
+#min_interval=0.1
+
+# interval at which the scheduler will check for jobs to be run
+#
+#resolution=0.1
+
+[security]
+
+# json configuration that defines users and their passwords - can point to a conf file
+#
+#users=etc/users.json
+
+# json configuration that maps users to permissions - can point to a conf file
+#
+#permissions=etc/permissions.json
+
+# [security.oauth]
+#
+## json configuration that maps users to permissions - can point to a conf file
+##
+##permissions=etc/permissions.json
+#
+## identify client that is making responses - used to get an access token
+##
+##client_id=407408718192.apps.googleusercontent.com
+#
+## url for validating access token
+##
+##validate_url=https=/www.googleapis.com/oauth2/v2/tokeninfo?access_token=%%s
 
 [web]
-server_thread_pool: 50
-request_show_tracebacks: true
 
-[environment]
-# location of custom extensions (blocks, services, etc)
-blocks.path: blocks
-components.path: niocore.components
+# web server max thread pool size
+#
+#server_thread_pool=50
 
-[components]
-# Class names comma separated list.
-exclusions: SNMPAgent
+# include error traceback when reporting failed requests
+#
+#request_show_tracebacks=false
 
-[ipc]
-request_time_to_live: 5.0
+[management_publisher]
+
+# pubsub topic key to publish on
+#
+#topics_key=type
+
+# pubsub topic value to publish on (for the given key)
+#
+#topics_value=management
+
+[rest]
 
-[server]
 # settings for the web server that will host the core REST API
-host: [[NIOHOST]]
-port: [[NIOPORT]]
+#
+
+# host and port to launch rest web server on
+#
+port=[[NIOPORT]]
+host=[[NIOHOST]]
```

Comment:
Could not find a reference to PUBLIC_IP (looks like a legacy setting)
Added comments to variables in nio.env